### ISSUE TITLE
6 Mastery points at level 20

### ISSUE THREAD

**Issue by RogertG** [WarEmu/WarBugs#11261]
*Issues should be unique. Check if someone else reported the issue first, and please don't report duplicates.*

Expected behavior and actual behavior:

At level 20 every char should have 5 mastery points to spend. I play an AM and at level 20 if someone specks in path of Asuryan, after dinging lvl 20, there is a (hopefully) visual bug.

If I try to respek (took me 1g to do it) I get back 5 points but 1 yellow mastery point remains in the Asuryan path.

If I try to take with the supposedly 6th point the first skill available in the Asuryan path, the game allows it but the skill is not usable.

Steps to reproduce the problem:

I really don't know... feel free to log into my account and try to respec from asuryan.

Screenshots/Videos or archive.org evidences:

https://we.tl/xiPrNev5ZH

* Complete rules for the bugtracker https://www.returnofreckoning.com/forum/viewtopic.php?f=52&t=10366

**Comment by RogertG** [comment]
Nevermind, it's not a bug. I mean I tested and tested 'till I figured it out that it's because of the armos set giving 1 level in Path of Asuryan. I just didn't knew that wearing a set that gives a level in a certain path appears in orange as if actually I'd spend a mastery point in that path.

Anyway, that 1 level given by a set should not grant the user to take an ability with, it should only strengthen the abilities in that path ~ at least that was the way they worked in the past.
